server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/StartAuditRound.java

```diff
@@ -140,7 +155,7 @@ public String startRoundOne(final Request the_request, final Response the_respon
             asm.stepEvent(COUNTY_START_AUDIT_EVENT);
             final ASMEvent audit_event;
             if (asm.currentState().equals(CountyDashboardState.COUNTY_AUDIT_UNDERWAY) &&
-                cdb.comparisonAudits().isEmpty()) {
+                (cdb.comparisonAudits().isEmpty() || cdb.estimatedBallotsToAudit() == 0)) {
```

Comment:
In other places, we are checking that the ballots to audit is `> 0`. Could the result of `cdb.estimatedBallotsToAudit()` ever be negative, or was that fixed when you removed the the three-loop algorithm?

Reply:
I will check on that when I check on the issue Neal is still having.